dashboard/app/models/authentication_option.rb

```diff
@@ -29,12 +29,32 @@ class AuthenticationOption < ApplicationRecord
   before_save :normalize_email, :hash_email, :remove_student_cleartext_email,
     :fill_authentication_id
 
+  OAUTH_CREDENTIAL_TYPES = [
+    CLEVER = 'clever',
+    FACEBOOK = 'facebook',
+    GOOGLE = 'google_oauth2',
+    POWERSCHOOL = 'powerschool',
+    QUIKCAMPS = 'lti_lti_prod_kids.qwikcamps.com',
```

Comment:
Providers we haven't tested and don't support migrating yet: Quikcamps, [support added June 2017](https://github.com/code-dot-org/code-dot-org/pull/15777), in use by at least 25 user accounts, but http://quikcamps.com/ can't be reached so not sure what the status of this is.